analytic_line_is_timesheet/models/account_analytic_line.py

```diff
@@ -18,7 +18,7 @@ def compute_timesheet_values(self):
     @api.model
     def create(self, vals):
         line = super(AccountAnalyticLine, self).create(vals)
-        if line.is_timesheet:
+        if line.is_timesheet and line.user_id:
```

Comment:
only the condition `line.is_timesheet` is required here.

in the case where

`
if line.is_timesheet:

    if not line.user_id:

        raise ValidationError(..)
`
    ...